<!doctype html>
<html>
  <head>
    <title>Page title</title>
    <link rel="stylesheet"  href="/lgsf-dashboard/assets/css/styles.css">
    <style>



    </style>

  </head>
  <body>



    <div class="ds-page">
      <a class="ds-skip-link" href="#main">skip to content</a>

      <header class="ds-header">
        <a class="ds-skip-link" href="#main">skip to content</a>
        <a class="ds-logo" href="/lgsf-dashboard/">
          <img src="https://DemocracyClub.github.io/design-system/images/logo_icon.svg" alt="" />
          <span>LGSF Logbooks</span>
        </a>

      </header>


      <main id="main" tabindex="-1" class="ds-stack">

        
<style>

    pre {
        height:400px;
        overflow-x: scroll;
        width:100%
    }
</style>




  <h2 id="2022-03-27-11-25">2022-03-27</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>1 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2022-03-27 11:25:24.649521</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2022-03-27 11:25:26.562457</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd></dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>None: Max retries exceeded with url: /cmis/CouncillorsAtoZ/tabid/39/ScreenMode/Ward/Default.aspx (Caused by None)</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[11:25:24] Fetching Scraper for: WAW                              handlers.py:22
           Begin attempting to scrape: WAW                        handlers.py:25
           Deleting existing data...                                 base.py:234
           Getting all files in WAW...                               base.py:186
[11:25:25] ...found 1 files in WAW                                   base.py:202
           Deleting batch no. 1 consisting of 1 files                base.py:211
[11:25:26] ...data deleted.                                          base.py:241
           Scraping from https://estates4.warwickdc.gov.uk/cmis/Counc base.py:40
           illorsAtoZ/tabid/39/ScreenMode/Ward/Default.aspx                     
           HTTPSConnectionPool(host='estates4.warwickdc.gov.uk',  handlers.py:34
           port=443): Max retries exceeded with url: /cmis/Counci               
           llorsAtoZ/tabid/39/ScreenMode/Ward/Default.aspx                      
           (Caused by NewConnectionError('<urllib3.connection.HTT               
           PSConnection object at 0x7f0d80e6ee80>: Failed to                    
           establish a new connection: [Errno -5] No address                    
           associated with hostname'))                                          
           Finished attempting to scrape: WAW                        base.py:319
</pre>

  <h2 id="2022-03-26-11-39">2022-03-26</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>2 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2022-03-26 11:39:03.580272</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2022-03-26 11:39:05.933788</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd></dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>None: Max retries exceeded with url: /cmis/CouncillorsAtoZ/tabid/39/ScreenMode/Ward/Default.aspx (Caused by None)</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[11:39:03] Fetching Scraper for: WAW                              handlers.py:22
           Begin attempting to scrape: WAW                        handlers.py:25
           Deleting existing data...                                 base.py:234
[11:39:04] Getting all files in WAW...                               base.py:186
           ...found 1 files in WAW                                   base.py:202
           Deleting batch no. 1 consisting of 1 files                base.py:211
[11:39:05] ...data deleted.                                          base.py:241
           Scraping from https://estates4.warwickdc.gov.uk/cmis/Counc base.py:40
           illorsAtoZ/tabid/39/ScreenMode/Ward/Default.aspx                     
           HTTPSConnectionPool(host='estates4.warwickdc.gov.uk',  handlers.py:34
           port=443): Max retries exceeded with url: /cmis/Counci               
           llorsAtoZ/tabid/39/ScreenMode/Ward/Default.aspx                      
           (Caused by NewConnectionError('<urllib3.connection.HTT               
           PSConnection object at 0x7f1e3f838040>: Failed to                    
           establish a new connection: [Errno -5] No address                    
           associated with hostname'))                                          
           Finished attempting to scrape: WAW                        base.py:319
</pre>

  <h2 id="2022-03-25-11-35">2022-03-25</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>8 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2022-03-25 11:35:01.969052</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2022-03-25 11:35:10.640954</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd></dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>An error occurred (ThrottlingException) when calling the CreateCommit operation (reached max retries: 4): Rate exceeded</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[11:35:01] Fetching Scraper for: WAW                              handlers.py:22
           Begin attempting to scrape: WAW                        handlers.py:25
[11:35:02] Deleting existing data...                                 base.py:234
           Getting all files in WAW...                               base.py:186
           ...found 1 files in WAW                                   base.py:202
           Deleting batch no. 1 consisting of 1 files                base.py:211
[11:35:10] An error occurred (ThrottlingException) when calling   handlers.py:34
           the CreateCommit operation (reached max retries: 4):                 
           Rate exceeded                                                        
           Finished attempting to scrape: WAW                        base.py:319
</pre>

  <h2 id="2022-03-24-11-34">2022-03-24</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>4 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2022-03-24 11:34:04.726732</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2022-03-24 11:34:09.015318</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd></dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>An error occurred (ThrottlingException) when calling the CreateCommit operation (reached max retries: 4): Rate exceeded</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[11:34:04] Fetching Scraper for: WAW                              handlers.py:22
           Begin attempting to scrape: WAW                        handlers.py:25
           Deleting existing data...                                 base.py:234
[11:34:05] Getting all files in WAW...                               base.py:186
           ...found 1 files in WAW                                   base.py:202
           Deleting batch no. 1 consisting of 1 files                base.py:211
[11:34:08] An error occurred (ThrottlingException) when calling   handlers.py:34
           the CreateCommit operation (reached max retries: 4):                 
           Rate exceeded                                                        
[11:34:09] Finished attempting to scrape: WAW                        base.py:319
</pre>

  <h2 id="2022-03-23-11-26">2022-03-23</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>5 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2022-03-23 11:26:56.433984</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2022-03-23 11:27:01.933536</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd></dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>An error occurred (ThrottlingException) when calling the CreateCommit operation (reached max retries: 4): Rate exceeded</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[11:26:56] Fetching Scraper for: WAW                              handlers.py:22
           Begin attempting to scrape: WAW                        handlers.py:25
           Deleting existing data...                                 base.py:234
           Getting all files in WAW...                               base.py:186
           ...found 1 files in WAW                                   base.py:202
           Deleting batch no. 1 consisting of 1 files                base.py:211
[11:27:01] An error occurred (ThrottlingException) when calling   handlers.py:34
           the CreateCommit operation (reached max retries: 4):                 
           Rate exceeded                                                        
           No new councillor data found.                             base.py:317
           Finished attempting to scrape: WAW                        base.py:319
</pre>

  <h2 id="2022-03-23-00-08">2022-03-23</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>11 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2022-03-23 00:08:36.579386</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2022-03-23 00:08:48.323671</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd></dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>An error occurred (ThrottlingException) when calling the CreateCommit operation (reached max retries: 4): Rate exceeded</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[00:08:36] Fetching Scraper for: WAW                              handlers.py:22
           Begin attempting to scrape: WAW                        handlers.py:25
           Deleting existing data...                                 base.py:234
           Getting all files in WAW...                               base.py:186
[00:08:37] ...found 1 files in WAW                                   base.py:202
           Deleting batch no. 1 consisting of 1 files                base.py:211
[00:08:48] An error occurred (ThrottlingException) when calling   handlers.py:34
           the CreateCommit operation (reached max retries: 4):                 
           Rate exceeded                                                        
           Finished attempting to scrape: WAW                        base.py:319
</pre>

  <h2 id="2022-03-21-12-18">2022-03-21</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>0 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2022-03-21 12:18:44.417657</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2022-03-21 12:18:45.411343</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd></dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>None: Max retries exceeded with url: /cmis/CouncillorsAtoZ/tabid/39/ScreenMode/Ward/Default.aspx (Caused by None)</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[12:18:44] Fetching Scraper for: WAW                              handlers.py:22
           Begin attempting to scrape: WAW                        handlers.py:25
           Deleting existing data...                                 base.py:234
           Getting all files in WAW...                               base.py:186
           WAW Does not exist                                        base.py:206
           ...no data to delete.                                     base.py:238
           Scraping from https://estates4.warwickdc.gov.uk/cmis/Counc base.py:40
           illorsAtoZ/tabid/39/ScreenMode/Ward/Default.aspx                     
[12:18:45] HTTPSConnectionPool(host='estates4.warwickdc.gov.uk',  handlers.py:34
           port=443): Max retries exceeded with url: /cmis/Counci               
           llorsAtoZ/tabid/39/ScreenMode/Ward/Default.aspx                      
           (Caused by NewConnectionError('<urllib3.connection.HTT               
           PSConnection object at 0x7f8f605ff640>: Failed to                    
           establish a new connection: [Errno -5] No address                    
           associated with hostname'))                                          
           Finished attempting to scrape: WAW                        base.py:319
</pre>

  <h2 id="2022-03-20-11-39">2022-03-20</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>2 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2022-03-20 11:39:18.388136</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2022-03-20 11:39:20.790371</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd></dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>None: Max retries exceeded with url: /cmis/CouncillorsAtoZ/tabid/39/ScreenMode/Ward/Default.aspx (Caused by None)</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[11:39:18] Fetching Scraper for: WAW                              handlers.py:22
           Begin attempting to scrape: WAW                        handlers.py:25
           Deleting existing data...                                 base.py:234
           Getting all files in WAW...                               base.py:186
           ...found 1 files in WAW                                   base.py:202
           Deleting batch no. 1 consisting of 1 files                base.py:211
[11:39:20] ...data deleted.                                          base.py:241
           Scraping from https://estates4.warwickdc.gov.uk/cmis/Counc base.py:40
           illorsAtoZ/tabid/39/ScreenMode/Ward/Default.aspx                     
           HTTPSConnectionPool(host='estates4.warwickdc.gov.uk',  handlers.py:34
           port=443): Max retries exceeded with url: /cmis/Counci               
           llorsAtoZ/tabid/39/ScreenMode/Ward/Default.aspx                      
           (Caused by NewConnectionError('<urllib3.connection.HTT               
           PSConnection object at 0x7fef9153f3d0>: Failed to                    
           establish a new connection: [Errno -5] No address                    
           associated with hostname'))                                          
           Finished attempting to scrape: WAW                        base.py:319
</pre>

  <h2 id="2022-03-19-11-49">2022-03-19</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>11 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2022-03-19 11:49:27.035093</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2022-03-19 11:49:38.901126</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd></dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>An error occurred (ThrottlingException) when calling the CreateCommit operation (reached max retries: 4): Rate exceeded</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[11:49:27] Fetching Scraper for: WAW                              handlers.py:22
           Begin attempting to scrape: WAW                        handlers.py:25
           Deleting existing data...                                 base.py:234
           Getting all files in WAW...                               base.py:186
[11:49:28] ...found 1 files in WAW                                   base.py:202
           Deleting batch no. 1 consisting of 1 files                base.py:211
[11:49:38] An error occurred (ThrottlingException) when calling   handlers.py:34
           the CreateCommit operation (reached max retries: 4):                 
           Rate exceeded                                                        
           No new councillor data found.                             base.py:317
           Finished attempting to scrape: WAW                        base.py:319
</pre>

  <h2 id="2022-03-18-11-22">2022-03-18</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>5 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2022-03-18 11:22:35.349248</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2022-03-18 11:22:41.203065</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd></dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>An error occurred (ThrottlingException) when calling the CreateCommit operation (reached max retries: 4): Rate exceeded</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[11:22:35] Fetching Scraper for: WAW                              handlers.py:22
           Begin attempting to scrape: WAW                        handlers.py:25
           Deleting existing data...                                 base.py:234
[11:22:36] Getting all files in WAW...                               base.py:186
           ...found 1 files in WAW                                   base.py:202
           Deleting batch no. 1 consisting of 1 files                base.py:211
[11:22:40] An error occurred (ThrottlingException) when calling   handlers.py:34
           the CreateCommit operation (reached max retries: 4):                 
           Rate exceeded                                                        
[11:22:41] No new councillor data found.                             base.py:317
           Finished attempting to scrape: WAW                        base.py:319
</pre>

  <h2 id="2022-03-17-11-42">2022-03-17</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>11 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2022-03-17 11:42:52.726466</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2022-03-17 11:43:04.347037</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd></dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>An error occurred (ThrottlingException) when calling the CreateCommit operation (reached max retries: 4): Rate exceeded</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[11:42:52] Fetching Scraper for: WAW                              handlers.py:22
           Begin attempting to scrape: WAW                        handlers.py:25
           Deleting existing data...                                 base.py:234
[11:42:53] Getting all files in WAW...                               base.py:186
           ...found 1 files in WAW                                   base.py:202
           Deleting batch no. 1 consisting of 1 files                base.py:211
[11:43:04] An error occurred (ThrottlingException) when calling   handlers.py:34
           the CreateCommit operation (reached max retries: 4):                 
           Rate exceeded                                                        
           No new councillor data found.                             base.py:317
           Finished attempting to scrape: WAW                        base.py:319
</pre>

  <h2 id="2022-03-16-11-56">2022-03-16</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>2 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2022-03-16 11:56:31.816842</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2022-03-16 11:56:34.108171</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd></dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>None: Max retries exceeded with url: /cmis/CouncillorsAtoZ/tabid/39/ScreenMode/Ward/Default.aspx (Caused by None)</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[11:56:31] Fetching Scraper for: WAW                              handlers.py:22
           Begin attempting to scrape: WAW                        handlers.py:25
           Deleting existing data...                                 base.py:234
[11:56:32] Getting all files in WAW...                               base.py:186
           ...found 1 files in WAW                                   base.py:202
           Deleting batch no. 1 consisting of 1 files                base.py:211
[11:56:33] ...data deleted.                                          base.py:241
           Scraping from https://estates4.warwickdc.gov.uk/cmis/Counc base.py:40
           illorsAtoZ/tabid/39/ScreenMode/Ward/Default.aspx                     
           HTTPSConnectionPool(host='estates4.warwickdc.gov.uk',  handlers.py:34
           port=443): Max retries exceeded with url: /cmis/Counci               
           llorsAtoZ/tabid/39/ScreenMode/Ward/Default.aspx                      
           (Caused by NewConnectionError('<urllib3.connection.HTT               
           PSConnection object at 0x7f26fa862130>: Failed to                    
           establish a new connection: [Errno -5] No address                    
           associated with hostname'))                                          
[11:56:34] Finished attempting to scrape: WAW                        base.py:319
</pre>

  <h2 id="2022-03-16-00-00">2022-03-16</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>2 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2022-03-16 00:00:12.991105</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2022-03-16 00:00:15.437752</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd></dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>None: Max retries exceeded with url: /cmis/CouncillorsAtoZ/tabid/39/ScreenMode/Ward/Default.aspx (Caused by None)</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[00:00:12] Fetching Scraper for: WAW                              handlers.py:22
[00:00:13] Begin attempting to scrape: WAW                        handlers.py:25
           Deleting existing data...                                 base.py:234
           Getting all files in WAW...                               base.py:186
           ...found 1 files in WAW                                   base.py:202
           Deleting batch no. 1 consisting of 1 files                base.py:211
[00:00:15] ...data deleted.                                          base.py:241
           Scraping from https://estates4.warwickdc.gov.uk/cmis/Counc base.py:40
           illorsAtoZ/tabid/39/ScreenMode/Ward/Default.aspx                     
           HTTPSConnectionPool(host='estates4.warwickdc.gov.uk',  handlers.py:34
           port=443): Max retries exceeded with url: /cmis/Counci               
           llorsAtoZ/tabid/39/ScreenMode/Ward/Default.aspx                      
           (Caused by NewConnectionError('<urllib3.connection.HTT               
           PSConnection object at 0x7f1cd3e1ce50>: Failed to                    
           establish a new connection: [Errno -5] No address                    
           associated with hostname'))                                          
           Finished attempting to scrape: WAW                        base.py:319
</pre>

  <h2 id="2022-03-14-11-14">2022-03-14</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>2 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2022-03-14 11:14:33.868506</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2022-03-14 11:14:36.152184</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd></dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>None: Max retries exceeded with url: /cmis/CouncillorsAtoZ/tabid/39/ScreenMode/Ward/Default.aspx (Caused by None)</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[11:14:33] Fetching Scraper for: WAW                              handlers.py:22
           Begin attempting to scrape: WAW                        handlers.py:25
           Deleting existing data...                                 base.py:234
[11:14:34] Getting all files in WAW...                               base.py:186
           ...found 1 files in WAW                                   base.py:202
           Deleting batch no. 1 consisting of 1 files                base.py:211
[11:14:35] ...data deleted.                                          base.py:241
           Scraping from https://estates4.warwickdc.gov.uk/cmis/Counc base.py:40
           illorsAtoZ/tabid/39/ScreenMode/Ward/Default.aspx                     
           HTTPSConnectionPool(host='estates4.warwickdc.gov.uk',  handlers.py:34
           port=443): Max retries exceeded with url: /cmis/Counci               
           llorsAtoZ/tabid/39/ScreenMode/Ward/Default.aspx                      
           (Caused by NewConnectionError('<urllib3.connection.HTT               
           PSConnection object at 0x7f5c77df91f0>: Failed to                    
           establish a new connection: [Errno -5] No address                    
           associated with hostname'))                                          
[11:14:36] Finished attempting to scrape: WAW                        base.py:319
</pre>

  <h2 id="2022-03-13-11-47">2022-03-13</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>2 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2022-03-13 11:47:21.256953</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2022-03-13 11:47:23.855431</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd></dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>None: Max retries exceeded with url: /cmis/CouncillorsAtoZ/tabid/39/ScreenMode/Ward/Default.aspx (Caused by None)</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[11:47:21] Fetching Scraper for: WAW                              handlers.py:22
           Begin attempting to scrape: WAW                        handlers.py:25
           Deleting existing data...                                 base.py:234
           Getting all files in WAW...                               base.py:186
           ...found 1 files in WAW                                   base.py:202
           Deleting batch no. 1 consisting of 1 files                base.py:211
[11:47:23] ...data deleted.                                          base.py:241
           Scraping from https://estates4.warwickdc.gov.uk/cmis/Counc base.py:40
           illorsAtoZ/tabid/39/ScreenMode/Ward/Default.aspx                     
           HTTPSConnectionPool(host='estates4.warwickdc.gov.uk',  handlers.py:34
           port=443): Max retries exceeded with url: /cmis/Counci               
           llorsAtoZ/tabid/39/ScreenMode/Ward/Default.aspx                      
           (Caused by NewConnectionError('<urllib3.connection.HTT               
           PSConnection object at 0x7f5960e07d60>: Failed to                    
           establish a new connection: [Errno -5] No address                    
           associated with hostname'))                                          
           Finished attempting to scrape: WAW                        base.py:319
</pre>

  <h2 id="2022-03-12-11-28">2022-03-12</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>0 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2022-03-12 11:28:14.551923</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2022-03-12 11:28:15.485258</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd></dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>None: Max retries exceeded with url: /cmis/CouncillorsAtoZ/tabid/39/ScreenMode/Ward/Default.aspx (Caused by None)</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[11:28:14] Fetching Scraper for: WAW                              handlers.py:22
           Begin attempting to scrape: WAW                        handlers.py:25
           Deleting existing data...                                 base.py:234
           Getting all files in WAW...                               base.py:186
           WAW Does not exist                                        base.py:206
[11:28:15] ...no data to delete.                                     base.py:238
           Scraping from https://estates4.warwickdc.gov.uk/cmis/Counc base.py:40
           illorsAtoZ/tabid/39/ScreenMode/Ward/Default.aspx                     
           HTTPSConnectionPool(host='estates4.warwickdc.gov.uk',  handlers.py:34
           port=443): Max retries exceeded with url: /cmis/Counci               
           llorsAtoZ/tabid/39/ScreenMode/Ward/Default.aspx                      
           (Caused by NewConnectionError('<urllib3.connection.HTT               
           PSConnection object at 0x7f34fac72040>: Failed to                    
           establish a new connection: [Errno -5] No address                    
           associated with hostname'))                                          
           Finished attempting to scrape: WAW                        base.py:319
</pre>

  <h2 id="2022-03-11-11-51">2022-03-11</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>0 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2022-03-11 11:51:44.633828</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2022-03-11 11:51:45.621943</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd></dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>None: Max retries exceeded with url: /cmis/CouncillorsAtoZ/tabid/39/ScreenMode/Ward/Default.aspx (Caused by None)</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[11:51:44] Fetching Scraper for: WAW                              handlers.py:22
           Begin attempting to scrape: WAW                        handlers.py:25
           Deleting existing data...                                 base.py:234
           Getting all files in WAW...                               base.py:186
[11:51:45] WAW Does not exist                                        base.py:206
           ...no data to delete.                                     base.py:238
           Scraping from https://estates4.warwickdc.gov.uk/cmis/Counc base.py:40
           illorsAtoZ/tabid/39/ScreenMode/Ward/Default.aspx                     
           HTTPSConnectionPool(host='estates4.warwickdc.gov.uk',  handlers.py:34
           port=443): Max retries exceeded with url: /cmis/Counci               
           llorsAtoZ/tabid/39/ScreenMode/Ward/Default.aspx                      
           (Caused by NewConnectionError('<urllib3.connection.HTT               
           PSConnection object at 0x7fe787a6d610>: Failed to                    
           establish a new connection: [Errno -5] No address                    
           associated with hostname'))                                          
           Finished attempting to scrape: WAW                        base.py:319
</pre>

  <h2 id="2022-03-10-11-16">2022-03-10</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>4 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2022-03-10 11:16:45.955459</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2022-03-10 11:16:50.100882</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd></dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>None: Max retries exceeded with url: /cmis/CouncillorsAtoZ/tabid/39/ScreenMode/Ward/Default.aspx (Caused by None)</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[11:16:45] Fetching Scraper for: WAW                              handlers.py:22
           Begin attempting to scrape: WAW                        handlers.py:25
[11:16:46] Deleting existing data...                                 base.py:234
           Getting all files in WAW...                               base.py:186
           ...found 1 files in WAW                                   base.py:202
           Deleting batch no. 1 consisting of 1 files                base.py:211
[11:16:49] ...data deleted.                                          base.py:241
           Scraping from https://estates4.warwickdc.gov.uk/cmis/Counc base.py:40
           illorsAtoZ/tabid/39/ScreenMode/Ward/Default.aspx                     
           HTTPSConnectionPool(host='estates4.warwickdc.gov.uk',  handlers.py:34
           port=443): Max retries exceeded with url: /cmis/Counci               
           llorsAtoZ/tabid/39/ScreenMode/Ward/Default.aspx                      
           (Caused by NewConnectionError('<urllib3.connection.HTT               
           PSConnection object at 0x7fa831b7a310>: Failed to                    
           establish a new connection: [Errno -5] No address                    
           associated with hostname'))                                          
[11:16:50] Finished attempting to scrape: WAW                        base.py:319
</pre>

  <h2 id="2022-03-09-11-31">2022-03-09</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>3 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2022-03-09 11:31:23.749324</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2022-03-09 11:31:27.362063</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd></dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>None: Max retries exceeded with url: /cmis/CouncillorsAtoZ/tabid/39/ScreenMode/Ward/Default.aspx (Caused by None)</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[11:31:23] Fetching Scraper for: WAW                              handlers.py:22
           Begin attempting to scrape: WAW                        handlers.py:25
           Deleting existing data...                                 base.py:234
[11:31:24] Getting all files in WAW...                               base.py:186
           ...found 1 files in WAW                                   base.py:202
           Deleting batch no. 1 consisting of 1 files                base.py:211
[11:31:27] ...data deleted.                                          base.py:241
           Scraping from https://estates4.warwickdc.gov.uk/cmis/Counc base.py:40
           illorsAtoZ/tabid/39/ScreenMode/Ward/Default.aspx                     
           HTTPSConnectionPool(host='estates4.warwickdc.gov.uk',  handlers.py:34
           port=443): Max retries exceeded with url: /cmis/Counci               
           llorsAtoZ/tabid/39/ScreenMode/Ward/Default.aspx                      
           (Caused by NewConnectionError('<urllib3.connection.HTT               
           PSConnection object at 0x7f7a2927acd0>: Failed to                    
           establish a new connection: [Errno -5] No address                    
           associated with hostname'))                                          
           Finished attempting to scrape: WAW                        base.py:319
</pre>

  <h2 id="2022-03-08-11-46">2022-03-08</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>2 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2022-03-08 11:46:36.383690</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2022-03-08 11:46:38.612789</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd></dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>None: Max retries exceeded with url: /cmis/CouncillorsAtoZ/tabid/39/ScreenMode/Ward/Default.aspx (Caused by None)</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[11:46:36] Fetching Scraper for: WAW                              handlers.py:22
           Begin attempting to scrape: WAW                        handlers.py:25
           Deleting existing data...                                 base.py:234
[11:46:37] Getting all files in WAW...                               base.py:186
           ...found 1 files in WAW                                   base.py:202
           Deleting batch no. 1 consisting of 1 files                base.py:211
[11:46:38] ...data deleted.                                          base.py:241
           Scraping from https://estates4.warwickdc.gov.uk/cmis/Counc base.py:40
           illorsAtoZ/tabid/39/ScreenMode/Ward/Default.aspx                     
           HTTPSConnectionPool(host='estates4.warwickdc.gov.uk',  handlers.py:34
           port=443): Max retries exceeded with url: /cmis/Counci               
           llorsAtoZ/tabid/39/ScreenMode/Ward/Default.aspx                      
           (Caused by NewConnectionError('<urllib3.connection.HTT               
           PSConnection object at 0x7f52e49b3640>: Failed to                    
           establish a new connection: [Errno -5] No address                    
           associated with hostname'))                                          
           Finished attempting to scrape: WAW                        base.py:319
</pre>

  <h2 id="2022-03-07-12-29">2022-03-07</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>2 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2022-03-07 12:29:15.539276</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2022-03-07 12:29:18.122216</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd></dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>None: Max retries exceeded with url: /cmis/CouncillorsAtoZ/tabid/39/ScreenMode/Ward/Default.aspx (Caused by None)</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[12:29:15] Fetching Scraper for: WAW                              handlers.py:22
           Begin attempting to scrape: WAW                        handlers.py:25
           Deleting existing data...                                 base.py:234
           Getting all files in WAW...                               base.py:186
[12:29:16] ...found 1 files in WAW                                   base.py:202
           Deleting batch no. 1 consisting of 1 files                base.py:211
[12:29:17] ...data deleted.                                          base.py:241
           Scraping from https://estates4.warwickdc.gov.uk/cmis/Counc base.py:40
           illorsAtoZ/tabid/39/ScreenMode/Ward/Default.aspx                     
           HTTPSConnectionPool(host='estates4.warwickdc.gov.uk',  handlers.py:34
           port=443): Max retries exceeded with url: /cmis/Counci               
           llorsAtoZ/tabid/39/ScreenMode/Ward/Default.aspx                      
           (Caused by NewConnectionError('<urllib3.connection.HTT               
           PSConnection object at 0x7f9b47095100>: Failed to                    
           establish a new connection: [Errno -5] No address                    
           associated with hostname'))                                          
[12:29:18] Finished attempting to scrape: WAW                        base.py:319
</pre>


      </main>
      <footer class="ds-footer">...</footer>
    </div>
  </body>
